src/components/admin/branchSettings/BranchFormModal.tsx

```diff
@@ -81,6 +90,11 @@ const BranchFormModal: React.FC<Props> = props => {
         enabled: gradingEnabled,
         group: gradingEnabled ? values.grading.group : undefined,
       },
+      automaticConfirmation: {
+        enabled: automaticConfirmationEnabled,
+        confirmationBranch: values.automaticConfirmation.confirmationBranch.id,
+        emails
```

Comment:
for this, can you follow the same syntax on the lines above? so like if `automaticConfirmationEnabled` is false, you set the other values to undefined